README.md

```diff
@@ -240,6 +284,21 @@ Response:
     < Content-Length: 2
     OK
 
+## Transports and Rules with dedicated timeouts
+
+It's transparent feature for end users for guarantee high availability and better transmission.
```

Comment:
Why do we speak about transparency here? Everything in a "client" section is transparent for end user, isn't it?

Reply:
Yes, it's redundant part.